app/services/api/group_keywords/keyword_creator.rb

```diff
@@ -0,0 +1,13 @@
+class Api::GroupKeywords::KeywordCreator
+    def initialize(group)
+        @group = group
+    end
+    def create_group_keywords(kw_ids)
```

Comment:
Small note on naming.  Look at the interface `Api::GroupKeywords::KeywordCreator#create_group_keywords`.  You could simplify this a little with `Api::Groups::KeywordCreator#call!` or something similar.  

Extra points -- For simple services like this, extending a `callable` module is an optional standard you could follow.  That would simplify the final call to just `Api::Groups::KeywordCreator.call(group_instance)`.  [Here's an article demonstrating that](https://bitcrowd.dev/improving-callable-service-objects-with-private-constructors-in-ruby).

Reply:
That's a good call, and I'm thinking that in some cases making a class method instead of having to instantiate a KeywordCreator object would look cleaner.